src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -1264,6 +2184,16 @@ private void testWkt(String geoWKT, String geoWKTSS) throws SQLException {
         return data;
     }
 
+    private void validateGeometryGeography(String geoWKT, byte[] geomCLR, byte[] geogCLR,
+            byte[] geoWKB) throws SQLServerException {
+        Geometry geomWKT = Geometry.deserialize(geomCLR);
+        Geography geogWKT = Geography.deserialize(geogCLR);
+        assertEquals(geoWKT, geomWKT.asTextZM());
+        assertEquals(geoWKT, geogWKT.asTextZM());
+        assertTrue(Arrays.equals(geomWKT.STAsBinary(), geoWKB));
```

Comment:
Here and in some the tests above, please make sure expected value is first.

Reply:
My bad, I forgot these ones.

Reply:
Actually, in this line, the expected value is coming first, right? I already swapped them this morning because you asked me to.

Reply:
`assertArrayEquals(geoWKB, geomWKT.STAsBinary());`

Reply:
geoWKB is the expected value

Reply:
I missed that one. I'll swap to assertArrayEquals as well.